lib/perl/Genome/Model/Event/Build/ReferenceAlignment/AlignReads.pm

```diff
@@ -488,7 +489,7 @@ sub _process_and_link_alignments_to_build {
     }
 
     for my $alignment (@alignments) {
-        my $link = $alignment->add_user(user => $build, label => 'uses');
+        my $link = $alignment->user(user => $build, label => 'uses');
```

Comment:
Where is the duplicate `add_user`?  I've gone across 3-4 classes and haven't found it...